### ISSUE TITLE
Bug/Missing UI Configuration: Unable to set Ollama host URL

### ISSUE THREAD

**Issue by sammcj** [intellibar/main#80]
I saw Intellibar was added to the list of apps that integrate with Ollama, went to try it out as it looks interesting, then got stuck as it doesn't seem there's anywhere to input your Ollama host URL?

<img width="671" alt="image" src="https://github.com/user-attachments/assets/463dfe67-4352-4b2f-9223-96e8e3b79dcc" />

I would have expected there to be a field or button to input the URL to your Ollama instance (along with an optional API key if Ollama is behind authentication), e.g. `https://ollama.my.internal.network` or `http://ollamaserver:11434`

**Comment by erusev** [comment]
Makes sense, we should add a way to set host URL, I'll see what we can do.

**Comment by astoilkov** [comment]
Hi @sammcj. I've started working on this. Can you tell me how do you set the `OLLAMA_HOST` config — env property or through `launchctl setenv`? Thanks!

**Comment by sammcj** [comment]
Howdy! Thanks for having a look at this.

Generally you set OLLAMA_HOST in the env for whatever client you're using only if it's a CLI tool (it would be set on the command line or in your bash/zsh profile), however if you're working with a GUI client these usually provide a configuration parameter in their UI / settings - otherwise the user would have to create a custom launchd job - or use the launchd setenv command which isn't always persistent.

Side note - you set OLLAMA_HOST in the launchd job for the Ollama server - it would try to listen on this address so that would only be set for things like using 0.0.0.0 instead of loopback or a custom port.

**Comment by astoilkov** [comment]
Aha, I think there's a misunderstanding here. IntelliBar allows the use of Ollama but you need to run it separately — most commonly by launching the Ollama desktop app or through the Terminal using `ollama serve`.

We use the default `OLLAMA_HOST` which is `127.0.0.1:11434`. What we can do to improve things is to read `OLLAMA_HOST` from the environment so we can support other hosts as well.

However, what I feel like it's happening is that you have the impression that IntelliBar will run Ollama which is not the case.

I correctly guessing what's happening?

**Comment by sammcj** [comment]
No I'm not expecting IntelliBar to run Ollama - I run Ollama on my server which is decked out with a good GPU, models etc...

If intellibar is always launched from the command line - reading the env var from the users shell makes sense.

If intellibar is to be launched graphically - it probably makes more sense just to let the user set the URL for their ollama server in the UI.
